auto_submit/test/requests/github_webhook_test_data.dart

```diff
@@ -240,6 +240,134 @@ String inProgressCheckRunsMock = '''{
   ]
 }''';
 
+String skippedCheckRunsMock = '''{
```

Comment:
Nit: const. Same for the following variables.

Reply:
Fixed. All global strings are now const.